SayTheirNames/Source/Controller/Home/HomeController.swift

```diff
@@ -161,4 +154,13 @@ extension HomeController: UICollectionViewDelegateFlowLayout {
             present(navigationController, animated: true, completion: nil)
         }
     }
+    
+    func collectionView(_ collectionView: UICollectionView, willDisplay cell: UICollectionViewCell, forItemAt indexPath: IndexPath) {
+        guard collectionView === peopleCollectionView else { return }
+        guard peopleDataSourceHelper.section(at: indexPath.section) == .main else { return }
+        
+        if indexPath.row == collectionView.numberOfItems(inSection: indexPath.section) - 1 {
+            viewModel.loadNextPage()
```

Comment:
Maybe we could move this to `collectionView:willDisplayCell:forItemAtIndexPath:` to start the loading of the next page as soon as possible?

Reply:
it's already been like that, so not sure what you mean 🤔  
now renamed the variable so this became outdated